Bottom: 42bcaace386ff46a10511b82928ad3474ee798b8
Top:    b0332e3d18e010cb32b7fa2f17fc6beb5afdf8ae
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 21:11:45 +0000

hid/gtk (GL): I think the polygon renderer works in mask mode now


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 286a362..158c37e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1454,11 +1454,8 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  common_fill_pcb_polygon (out->fgGC, &polygon, screen);
+  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
-  /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  hid_draw_fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  hid_draw_fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   hid_draw_use_mask (&ghid_graphics, HID_MASK_OFF);
